src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -254,6 +254,9 @@ def get_deployment_operations(client, resource_group_name, deployment_name, oper
 def list_resources(resource_group_name=None, resource_provider_namespace=None,
                    resource_type=None, name=None, tag=None, location=None):
     rcf = _resource_client_factory()
+
+    rcf.resource_groups.get(resource_group_name)
```

Comment:
Since resource group is optional, you need a conditional here or you will get "'resource_group_name' cannot be None" if it is omitted. I suspect you've got a test failure on that.